Src/VsSpecific/VsSpecific/CSharpScriptGlobals.cs

```diff
@@ -11,14 +11,16 @@ public class CSharpScriptGlobals
         public LineRangeSpecifier LineRange { get; }
         public bool IsScriptLocal { get; } = false;
         public IVim Vim { get; } = null;
+        public IVimBuffer VimBuffer { get; } = null;
 
-        public CSharpScriptGlobals(CallInfo callInfo, IVim vim)
+        public CSharpScriptGlobals(CallInfo callInfo, IVim vim, IVimBuffer vimBuffer)

```

Comment:
Functions that take an `IVimBuffer` argument don't also need an `IVim` argument because every `IVimBuffer` contains a reference to `IVim`.

Reply:
Thank you for the review.
I deleted unnecessary arguments.